src/pages/messages/TasksPage.tsx

```diff
@@ -105,6 +100,7 @@ const TasksPage = () => {
   const searchParams = new URLSearchParams(history.location.search);
 
   const queryParam = searchParams.get(TicketSearchParam.Query);
+  const assigneeParam = searchParams.get(TicketSearchParam.Assignee);
```

Comment:
Disse 2 brukes nå likt for både tickets og NVI. Ikke optimalt, da NVI har sin egen `NviCandidatesSearchParam.Assignee` som heller burde brukes. Men det er ikke noe praktisk problem heller, så tenker jeg lar den ligge til menyen her (forhåpentligvis) splittes i 2 komponenter; en for tickets, og en for NVI